Core/Main/PTMagic.cs

```diff
@@ -1648,11 +1648,6 @@ private void ApplySingleMarketSettings()
                                                   orderby m.Market
                                                   select m;
 
```

Comment:
Should remove lines 1642-1649 as well as we don't need the excludedMarkets if not outputting them to the log.